### ISSUE TITLE
Could you add visual studio 2010 files here?

### ISSUE THREAD

**Issue by greyltc** [BrianGladman/mpfr#4]

**Comment by greyltc** [comment]
I guess I had hoped that you were the one that created the VS2015 build files here. Where did those come from? Could VS2010 build files possibly come from that place too?

**Comment by BrianGladman** [comment]
Yes, I created the VS2015 build files and I maintain them here for the current version of Visual Studio.  I have maintained an MPFR build for the past 7 years and I do have old build files for VS2010 on my web site at: http://gladman.me.uk/computing/gmp4win.php    However, these are now very old and will need major revision in order to build the current version of MPFR.  They might, however, give you a start and you might  be able to update them by comparing them with my VS2015 build files to identify the changes needed (This will depend on whether Microsoft has changed its file format between VS2010 and VS2015).   Good luck - come back if you need any advice.
